src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala

```diff
@@ -562,4 +585,99 @@ class JobStats @Inject()(clusterBuilder: Option[Cluster.Builder], config: Cassan
       ConsistencyLevel.valueOf(config.cassandraConsistency())
     }
   }
+
+  private def pushNotificationToHook(hookurl: String,
+                                     port: Int,
+                                     id: Option[String],
+                                     timestamp: Option[java.util.Date],
+                                     jobName: Option[String],
+                                     jobOwner: Option[String],
+                                     jobSchedule: Option[String],
+                                     jobParents: Option[java.util.Set[String]],
+                                     taskState: Option[String],
+                                     slaveId: Option[String],
+                                     message: Option[String],
+                                     attempt: Option[Integer],
+                                     isFailure: Option[Boolean]) = {
+    val jsonBuffer = new StringWriter
+    val factory = new JsonFactory()
+    val generator = factory.createGenerator(jsonBuffer)
+
+    // Create the payload
+    generator.writeStartObject()
+
+    generator.writeStringField("taskId", id.get);
+    generator.writeStringField("ts", timestamp.get.toString());
+    generator.writeStringField("job_name", jobName.get);
+
+    jobOwner match {
+      case Some(jo: String) => generator.writeStringField("job_owner", jo)
+      case _ =>
+    }
+    jobSchedule match {
+      case Some(js: String) => generator.writeStringField("job_schedule", js)
+      case _ =>
+    }
+    jobParents match {
+      case Some(jp: java.util.Set[String]) => generator.writeStringField("job_parents", jp.toString())
+      case _ =>
+    }
+    taskState match {
+      case Some(ts: String) => generator.writeStringField("task_state", ts)
+      case _ =>
+    }
+    slaveId match {
+      case Some(s: String) => generator.writeStringField("slave_id", s)
+      case _ =>
+    }
+    message match {
+      case Some(m: String) => generator.writeStringField("message", m)
+      case _ =>
+    }
+    attempt match {
+      case Some(a: Integer) => generator.writeStringField("attempt", a.toString())
+      case _ =>
+    }
+    isFailure match {
```

Comment:
This does match the cassandra format, but it seems odd we use the negation isFailure over getSuccess. Out of scope for PR, just a thought.